internal/pkg/manifest/validate.go

```diff
@@ -204,9 +204,9 @@ func (b BackendService) validate() error {
 	}); err != nil {
 		return fmt.Errorf("validate HTTP load balancer target: %w", err)
 	}
-	if b.Network.Connect.Enabled() {
+	if b.Network.Connect.Alias != nil {
 		if b.RoutingRule.GetTargetContainer() == nil && b.ImageConfig.Port == nil {
-			return fmt.Errorf(`cannot enable "network.connect" when no port exposed`)
+			return fmt.Errorf(`cannot set "network.connect.alias" when no ports are exposed`)
```

Comment:
shouldn't this error also happen on `network.connect = true`? or no?

Reply:
nope only for `alias`. because they can still turn SC with no exposed endpoint so that it is a client-only service.